tests/test_delaunay.py

```diff
@@ -3,9 +3,12 @@
 from shapely.geometry import Polygon, LineString, Point
 from shapely.ops import triangulate
 from shapely.geos import geos_version
+import pytest
+
+
+pytestmark = pytest.mark.skipif(geos_version < (3, 4, 0), 
+                 reason='Delaunay triangulation not supported')
```

Comment:
This can actually be removed altogether, since we don't support those old GEOS versions anymore.